### DISCUSSION TITLE
0.5.0 Announcement

### DISCUSSION THREAD

**Discussion by gmcheems** [gmcheems/gmcheems#14] [Announcements]
It is with great pleasure that I announce the release of GM Cheems 0.5.0!!
This is an important release because GM Cheems is no longer a userscript; it is now a Chrome extension!
As a result of this change, **Lichess is now supported**!!!

## Background

GM Cheems started out as a userscript, and while userscripts are quite powerful, unfortunately, I ran into many technical limitations. One thing led to another, and I ultimately decided that it had to be converted to a Chrome extension. This allows GM Cheems to do things it could not do before, such as supporting Lichess. I'm excited about this as it opens up a lot of possibilities and I can't wait to see what more we'll be able to do in the future (looking at you, Stockfish 15 :slightly_smiling_face:)!

## Community
We now have a [Discord server](https://discord.gmcheems.com) and [YouTube channel](https://www.youtube.com/channel/UCLjpjeIBNzwpCIGPrUCdDtw).

## Pricing Update
0.3.0 will be the last free version of GM Cheems. Starting with 0.5.0, GM Cheems will require a license which can be purchased [here](https://www.buymeacoffee.com/gmcheems). ~~(Item will show up once extension is available for download)~~

While I understand some might be upset, the income I make will go directly to making GM Cheems better, adding more features and content.

## So where do I get it?
~~The extension has been submitted to the Chrome webstore. Once it's been approved I will post the link here.~~
https://chrome.google.com/webstore/detail/gm-cheems/bmkmigdaklnhklmiialljbpjiedmnifh?hl=en&authuser=0

~~cheems

**Comment by 007sg** [comment]
Does version 0.5.0 already support Chess960?
Is there any payment without credit card?
